back/app/policies/baskets_idea_policy.rb

```diff
@@ -19,18 +19,18 @@ def show?
   end
 
   def create?
-    BasketPolicy.new(user, record.basket).update?
+    BasketPolicy.new(user, record.basket).create?
```

Comment:
I would have a preference for `update?` because it feels like adding an idea to a basket is similar to editing a basket.